tests/tests_cib_rsc_op_defaults.yml

```diff
@@ -0,0 +1,167 @@
+          attrs:
+            - name: rsc-key-5
+              value: rsc-val-5
+            - name: rsc-key-6
+              value: rsc-val-6
+    ha_cluster_resource_operation_defaults:
+      meta_attrs:
+        - attrs:
+            - name: op-key-1
+              value: op-val-1
+        - attrs:
+            - name: op-key-2
+              value: op-val-2
+          id: custom_id_op
+        - attrs:
+            - name: op-key-3
+              value: op-val-3
+          score: 10
+        - attrs:
+            - name: op-key-4
+              value: op-val-4
+          rule: "resource ocf:pacemaker:"
+        - id: without_attrs_op
+          score: 20
+          rule: "resource :pacemaker: or op monitor"
+        - id: all_options_op
+          rule: resource ocf:pacemaker:test
+          score: INFINITY
+          attrs:
+            - name: op-key-5
+              value: op-val-5
+            - name: op-key-6
+              value: op-val-6
+  tasks:
+    - name: Run test
+      tags: tests::verify
+      block:
+        - name: Set up test environment
+          include_role:
+            name: linux-system-roles.ha_cluster
+            tasks_from: test_setup.yml
+
+        - name: Run HA Cluster role
+          include_role:
+            name: linux-system-roles.ha_cluster
+            public: true
+
+        - name: Verify resource defaults
+          vars:
+            __test_expected_lines:
+              - "Meta Attrs: rsc_defaults-meta_attributes"
+              - "  rsc-key-1=rsc-val-1"
+              - "Meta Attrs: custom_id_rsc"
+              - "  rsc-key-2=rsc-val-2"
+              - "Meta Attrs: rsc_defaults-meta_attributes-1 score=10"
+              - "  rsc-key-3=rsc-val-3"
+              - "Meta Attrs: rsc_defaults-meta_attributes-2"
+              - "  rsc-key-4=rsc-val-4"
+              - "  Rule: boolean-op=and score=INFINITY"
+              - "    Expression: resource ocf:pacemaker:"
+              - "Meta Attrs: without_attrs_rsc score=20"
+              - "  Rule: boolean-op=or score=INFINITY"
+              - "    Expression: resource :pacemaker:"
+              - "    Expression: resource :heartbeat:"
+              - "Meta Attrs: all_options_rsc score=INFINITY"
+              - "  rsc-key-5=rsc-val-5"
+              - "  rsc-key-6=rsc-val-6"
+              - "  Rule: boolean-op=and score=INFINITY"
+              - "    Expression: resource ocf:pacemaker:test"
+          block:
+            - name: Fetch resource defaults configuration from the cluster
+              command:
+                cmd: pcs resource defaults
+              register: __test_pcs_rsc_defaults_config
+              changed_when: false
+
+            - name: Print real resource defaults configuration
+              debug:
+                var: __test_pcs_rsc_defaults_config
+
+            - name: Print expected resource defaults configuration
+              debug:
+                var: __test_expected_lines | select | list
```

Comment:
do you need the `select` here?  afaict, all of the expected lines have a non-null and non-empty value

Reply:
You are right, it's not needed. I removed it.